src/windows-emulator/windows_emulator.cpp

```diff
@@ -448,9 +448,19 @@ void windows_emulator::setup_hooks()
     });
 
     this->emu().hook_instruction(x64_hookable_instructions::rdtsc, [&] {
-        const auto instructions = this->executed_instructions_;
-        this->emu().reg(x64_register::rax, instructions & 0xFFFFFFFF);
-        this->emu().reg(x64_register::rdx, (instructions >> 32) & 0xFFFFFFFF);
+        uint64_t ticks = this->clock_->timestamp_counter();
+        static uint64_t fake_ticks = ticks;
+        static uint64_t prev_ticks = 0;
```

Comment:
don't do that. there can be multiple instances of the emulator with different clocks. i don't think there is a need to store the previous ticks.
just write ticks into rax/rdx

Reply:
I will do that after some testing. 
If we do need to keep the previous time, would thread_local be an appropriate solution?